### PR TITLE
split relationship on the basis of selector w/signoff

### LINKED ISSUE #11123: [Models] Split out relationships into indivitual files and respective model folders.
### Current Situation

Currently, in the Meshery repository under the ```server/meshmodel``` directory, relationships between models are defined within the ```relationship.json``` file. This file contains all relationships in a centralized manner, which can make it challenging to manage and update specific relationships effectively.

### Proposed Change

To improve maintainability and clarity, it is proposed to split the data in ```relationship.json``` based on selectors and place them within their respective model folders.

---

### General Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### Understanding Meshery Models

- 🖥 [Models Overview](https://docs.google.com/presentation/d/1SQMfyu5shjpGKlYONdVzOtd7UYTgLWBcgUvHMLCZ2tY/edit#slide=id.g226f5de5180_19_259)
- 👨‍💻[Models Repository](https://github.com/meshery/meshery/tree/master/server/meshmodel)

### Contributing to Meshery Models

- https://docs.meshery.io/project/contributing/contributing-models

### Contributing to Meshery Relationships

- https://docs.meshery.io/project/contributing/contributing-relationships

  Review a prior pull request as an example of how to define a Relationships:   https://github.com/meshery/meshery/pull/9880/files

### PR DESCRIPTION
**Pull request by Jougan-0**
**Notes for Reviewers**

This PR fixes #11123 
```PR do not Merge```
Relationships before 14 now 19.
Migrated One relationship to istio-base from kubernetes

From Meshery Shapes to 
      1. Meshery Core
      2. Meshery Flowchart
      3. GCP
      4. AWS

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Jougan-0 what tests did you do to verify the functionality here?

**Review by leecalcote — APPROVED** [review]

**Comment by Jougan-0** [comment]
> @Jougan-0 what tests did you do to verify the functionality here?

@leecalcote  None of the tests have been conducted so far due to a bug with my account, which is now resolved. The only confirmation I have is that the registrations are taking place. With my account fixed, I will proceed with the testing. I'm not entirely certain about the correct method to follow for testing, but I will attempt to figure it out. If I encounter any issues, I will ask for help.

**Comment by leecalcote** [comment]
Thanks, be sure to use a label like "pr/draft" or add a similar note in the future. So that merge doesn't happen under the assumption that tests were diligently performed. 

Testing: just try to establish on of the relationships between these components using Designer.

**Review by leecalcote — CHANGES_REQUESTED** [review]
This PR only accomplishes a portion of what the issue calls for.

`server/meshmodel/kubernetes/v1.25.2/v1.0.0/relationships/hierarchical_inventory.json` is still entirely too long.

**Comment by Jougan-0** [comment]
[Docs](https://docs.google.com/document/d/13yauA29yTlr7WlOEv8R_08b1ZNhkRNHxFvZpK35f4_M/edit) issues.

**Comment by leecalcote** [comment]
Are docs issue the only outstanding item preventing merge?

**Review by leecalcote — APPROVED** [review]

**Comment by Jougan-0** [comment]
> Are docs issue the only outstanding item preventing merge?

Yes, docs issues are the items and they are quite a bit of a challenge this pr was updated so that uzair can take a look as the relationship evaluation on the client side is not correctly detecting the multiple relationship file.

**Comment by vishalvivekm** [comment]
@Jougan-0 Thank you for contributions.
Let's discuss this in Meshery Dev meeting on Wednesday. Adding this as an agenda item....
Meeting minutes: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#
Meeting link: https://meet.layer5.io/meshery

**Comment by leecalcote** [comment]
@innocentrda, it would be helpful if you can checkout this PR, build the Server, and verify a few of the relationships.

File: server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyFilter.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "schemaVersion": "core.meshery.io/v1alpha2",
+    "version": "v1.0.0",
+    "kind": "Hierarchical",
+    "metadata": {
+      "description": "A hierarchical inventory relationship in which the configuration of (parent) component is patched with the configuration of other (child) component. Eg: The configuration of the EnvoyFilter (parent) component is patched with the configuration as received from WASMFilter (child) component."
+    },
+    "model": {
+      "schemaVersion": "core.meshery.io/v1beta1",
+      "version": "v1.0.0",
+      "name": "istio-base",
```
Comment (leecalcote) [inline server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyFilter.json:11]: Good. 👍

File: server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyFilter.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+      "model": {
+        "version": "v1.16.0"
+      },
+      "displayName": "Istio",
+      "category": {
```
Comment (leecalcote) [inline server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyFilter.json:16]: Soon we will eliminate this duplication of information. Who will simply be a ref to the model.

File: server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyPlugin.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
+{
+    "schemaVersion": "core.meshery.io/v1alpha2",
```
Comment (leecalcote) [inline server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyPlugin.json:2]: I propose that we move this to:

```
"schemaVersion": "core.meshery.io/v1alpha2",

Comment (Jougan-0) [reply server/meshmodel/istio-base/1.16.0/v1.0.0/relationships/hierarchical_inventory_WASMFilterToEnvoyPlugin.json:2]: Surely, @leecalcote I saw the recent changes made by Uzair to the schemaVersion I'll be updating those refs.

**Pull request closed without merging**